pages/printers/index.tsx

```diff
@@ -3,7 +3,7 @@ import { gql, useQuery } from "@apollo/client";
 import { NextPage } from "next";
 import { useEffect, useState } from "react";
 import { tableStyle } from "../../styles/mui-datagrid";
-import { getStatusIcon } from "../../utils/statusIcon";
+import { getStatusIcon } from "../../Utils/statusIcon";
```

Comment:
Is the capital `U` intentional? Could we stick to lowercase, please?

Reply:
Yepp, of course. I think I created utils on two branches parallel, on one of them with capital by mistake....